Micro8085

ahjelm opened the pull request:
Pull Request: add new target 'micro8085' to z88dk target list. Anders Hjelm, ahjelm67@gmail.com

suborb commented:
@ahjelm thank you for this. As I mentioned, I'll add the sanity check builds etc

I've started a wiki page here: https://github.com/z88dk/z88dk/wiki/Platform--Micro85 if you don't mind correcting/adding any basic information I've missed then that would be appreciated - the main point of these pages is to give instructions on how to get things running/any additional library support.

ahjelm commented:
Hi,
Thanks! I've added a few things to the wiki page and also changed it into:
https://github.com/z88dk/z88dk/wiki/Platform--Micro8085 (typo missing the
number '80').
Best Regards
Anders


Den sön 12 dec. 2021 kl 17:52 skrev suborb ***@***.***>:

> @ahjelm <https://github.com/ahjelm> thank you for this. As I mentioned,
> I'll add the sanity check builds etc
>
> I've started a wiki page here:
> https://github.com/z88dk/z88dk/wiki/Platform--Micro85 if you don't mind
> correcting/adding any basic information I've missed then that would be
> appreciated - the main point of these pages is to give instructions on how
> to get things running/any additional library support.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/z88dk/z88dk/pull/1900#issuecomment-991931904>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AWOPKCO7DFYGW66HBXSVH3TUQTHLZANCNFSM5JPXVRBA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>
